examples/undo-tree/index.jsx

```diff
@@ -0,0 +1,53 @@
+import Microcosm from 'Microcosm'
+import React from 'react'
+import Drawing from './components/drawing'
+
+class UndoTree extends Microcosm {
+
+  shouldHistoryKeep(transaction) {
+    return true
+  }
```

Comment:
sounds like a good name to me. whatever the name though, this logic is not trivial and deserves a good writeup in the readme i believe.